### ISSUE TITLE
compile app directly for darling

### ISSUE THREAD

**Issue by schriftgestalt** [darlinghq/darling#847]
Darling is trying to run native macOS apps on Linux. Is there any interest in supporting developers trying to recompile their app directly for Linux. I wonder because you revived the old cocotron project. I never got it to work (the installation instruction are quite confusing and always outdated).

**Comment by LubosD** [comment]
No, because for us there's simply no benefit in doing so. Even Darling's own binaries are complied for macOS (and not for Linux).

Making Apple's frameworks available for Linux apps is a project for someone else.

**Comment by bugaevc** [comment]
But if there's interest in making Darling Cocotron work on Linux/Windows without the rest of Darling, we'd be open to accept contributions making that work again 🙂 Some people have talked about doing that on our Discord, but nobody actually did anything, so...

**Comment by schriftgestalt** [comment]
I would do it but I don’t understand it enough. I could pay someone to work on that if that would help.

**Comment by HinTak** [comment]
How far is openstep from cocoa?

**Comment by HinTak** [comment]
@schriftgestalt if what you are thinking is what I think, the first thing to try is to see whether the mac os version runs on darling or not. Then we know what is missing etc.

**Comment by HinTak** [comment]
@schriftgestalt https://github.com/darlinghq/darling-cocotron/pull/13 - I suspect you probably need some part of the unfinished pull :-)
